src/components/general/inputs/phone-number-input.js

```diff
@@ -56,9 +56,9 @@ class PhoneNumberInput extends Component {
   }
 
   onKeyDown(evt) {
-    // todo: for enter key
+    // todo: for enter key, investigate
     if(evt.key === 'Enter'){
-      this.props.onSubmit(this.state.phone);
+      //this.props.onSubmit(this.state.phone);
```

Comment:
whats wrong with this? seems to be working fine for me and there was another ticket that would be undone by commenting it out

Reply:
It broke, wanted to unblock for launch. It doesn't work for me. I'll try again